pkg/restore/interfaces.go

```diff
@@ -87,23 +84,28 @@ type appsV1Interface interface {
 //nolint:unused
 //goland:noinspection GoUnusedType
 type coreV1Interface interface {
-	v12.CoreV1Interface
+	corev1.CoreV1Interface
 }
 
 //nolint:unused
 //goland:noinspection GoUnusedType
-type ecosystemV1Alpha1Interface interface {
-	ecosystem.V1Alpha1Interface
+type configMapInterface interface {
+	corev1.ConfigMapInterface
 }
 
 //nolint:unused
 //goland:noinspection GoUnusedType
-type configurationContext interface {
-	registry.ConfigurationContext
+type ecosystemV1Alpha1Interface interface {
+	ecosystem.V1Alpha1Interface
 }
 
 //nolint:unused
 //goland:noinspection GoUnusedType
 type restoreProvider interface {
 	provider.Provider
 }
+
+type globalConfigRepository interface {
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>type `globalConfigRepository` is unused
<!-- __reviewdog__:ChA1NTFmOTAzNGE2ZWIyMmE1EgpjaGVja3N0eWxl -->